modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -116,15 +118,34 @@ function createDeck({
     // loading custom library is async, some layers might not be convertable before custom library loads
     const layerToLoad = missingLayers(oldLayers, convertedLayers);
     const getTooltip = makeTooltip(tooltip);
-
-    deckgl = new deck.DeckGL({
-      ...props,
-      map: mapboxgl,
-      mapboxApiAccessToken: mapboxApiKey,
-      onClick: handleClick,
-      getTooltip,
-      container
-    });
+    const {mapProvider} = props;
+    if (mapProvider === 'mapbox') {
+      deckgl = new deck.DeckGL({
+        ...props,
+        map: mapboxgl,
+        mapboxApiAccessToken: mapboxApiKey,
+        onClick: handleClick,
+        getTooltip,
+        container
+      });
+    } else if (mapProvider === 'google_maps') {
```

Comment:
Q: Provider = `google_maps` or just `google`?  

I suppose `google_maps` is more consistent with `googleMapsKey` etc